client/my-sites/stats/site.jsx

```diff
@@ -382,6 +408,7 @@ export default connect(
 			siteId && isJetpack && isJetpackModuleActive( state, siteId, 'stats' ) === false;
 		return {
 			isJetpack,
+			isPrivate: isPrivateSite( state, siteId ),
```

Comment:
It may be clearer to call this something like `isSitePrivate` -- I don't think `isPrivate` is sufficiently specific.